addons/mail/static/src/core/common/message_reactions_model.js

```diff
@@ -21,33 +21,9 @@ export class MessageReactions extends Record {
             /** @type {import("models").MessageReactions} */
             reaction = this.preinsert(data);
         }
-        const personasToUnlink = new Set();
-        const alreadyKnownPersonaIds = new Set(reaction.personas.map((p) => p.localId));
-        for (const rawPartner of data.partners) {
-            const [command, partnerData] = Array.isArray(rawPartner)
-                ? rawPartner
-                : ["ADD", rawPartner];
-            const persona = this.store.Persona.insert({ ...partnerData, type: "partner" });
-            if (command === "ADD" && !alreadyKnownPersonaIds.has(persona.localId)) {
-                reaction.personas.push(persona);
-            } else if (command !== "ADD") {
-                personasToUnlink.add(persona.localId);
-            }
-        }
-        for (const rawGuest of data.guests) {
-            const [command, guestData] = Array.isArray(rawGuest) ? rawGuest : ["ADD", rawGuest];
-            const persona = this.store.Persona.insert({ ...guestData, type: "guest" });
-            if (command === "ADD" && !alreadyKnownPersonaIds.has(persona.localId)) {
-                reaction.personas.push(persona);
-            } else if (command !== "ADD") {
-                personasToUnlink.add(persona.localId);
-            }
-        }
         Object.assign(reaction, {
             count: data.count,
-            content: data.content,
-            message: data.message,
-            personas: reaction.personas.filter((p) => !personasToUnlink.has(p.localId)),
+            personas: data.personas,
```

Comment:
not yet possible to completely assign data?

Reply:
Probably yes, will do the change now